src/GeoPoint.php

```diff
@@ -32,30 +32,24 @@ class GeoPoint implements JsonSerializable
      * @param float $lat Latitude
      * @param float $lng Longitude
      */
-    public function __construct($lat, $lng)
+    public function __construct($lat = 0, $lng = 0)
```

Comment:
And also, we had another bug in this point. Below test should be passed. Because, it means really 0, 0 point of world. 

```
        $point = new GeoPoint(0, 0);
        $this->assertEquals('0.0 , 0.0', (string) $point);
```